istio-installation.yaml

```diff
@@ -1,11 +1,15 @@
-apiVersion: istio.openshift.com/v1alpha3
-kind: ControlPlane
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
 metadata:
   name: istio-installation
   namespace: istio-system
 spec:
   istio:
     global:
+      # the Minishift VM runs CentOS 7, so we can't use the default UBI 8 init container
```

Comment:
Should `UBI 8` be replaced with `Universal Base Image` (I had to google to find out what UBI was :-))

Reply:
I expect this to be gone with Maistra 0.12, so I didn't give it much thought :-) Will fix.